SampleScripts/PowerManagement-WorkloadDomain.ps1

```diff
@@ -254,6 +254,42 @@ Try {
         [Array]$vcfvms = @()
         [Array]$vcfvms += ($vcServer.fqdn).Split(".")[0]
 
+        # Gather VxRail Manager Details for the VI Workload Domain, if exists
+        if ($PsBoundParameters.ContainsKey("shutdown")) {
+            $vxRailCred = (Get-VCFCredential | Where-Object { $_.resource.resourceType -eq "VXRAIL_MANAGER" -and $_.resource.domainName -eq ($workloadDomain.name) -and $_.username -eq "root" })
+            if ($vxRailCred -ne $null)
+            {
+			    # Connecting to VC to get VxRail VM name
```

Comment:
```suggestion
			    # Connecting to vCenter Server to get the VxRail Manager virtual machine name.
```

Reply:
addressed in commit f8c5a69ef84a1a40675a91879ad0a67701fba108